bundles/org.openhab.binding.volvooncall/src/main/java/org/openhab/binding/volvooncall/internal/dto/Status.java

```diff
@@ -63,31 +63,31 @@
     public String serviceWarningStatus = "";
     private @NonNullByDefault({}) List<Object> bulbFailures;
 
-    public Optional<WindowsStatus> getWindows() {
+    public @NonNullByDefault({}) Optional<WindowsStatus> getWindows() {
         return Optional.ofNullable(windows);
     }
 
-    public Optional<DoorsStatus> getDoors() {
+    public @NonNullByDefault({}) Optional<DoorsStatus> getDoors() {
```

Comment:
idem

Reply:
Please remove the uneeded annotations

Reply:
I thought we had decided that it was better for new developers to not have errors by default?

Reply:
My bad, I misread your message.

Just rebuilt my dev environment, and it seems the EEA worked this time, so my IDE is now also clean.